src/util/PathUtil.cpp:214:28: error: ‘relative’ is not a member of ‘fs’

**Affects versions :**
 - OS: linux
 - Version of Xournal++: master, commit 45184492057387fde4319358e8c2dc4c7ae5b80b
 - compiled from git
 -  gcc (Ubuntu 7.5.0-3ubuntu1~18.04) 7.5.0


**Describe the bug**
```
mkdir build; cd build; cmake ../
$ make
[  1%] Build translations.
[ 21%] Built target translations
[ 21%] Building CXX object src/util/CMakeFiles/util.dir/PathUtil.cpp.o
/home/bertossa/xournalpp/src/util/PathUtil.cpp: In function ‘bool Util::isChild(const std::experimental::filesystem::v1::__cxx11::path&, const std::experimental::filesystem::v1::__cxx11::path&)’:
/home/bertossa/xournalpp/src/util/PathUtil.cpp:214:28: error: ‘relative’ is not a member of ‘fs’
            *std::begin(fs::relative(path, base)) != "..";
                            ^~~~~~~~
src/util/CMakeFiles/util.dir/build.make:206: recipe for target 'src/util/CMakeFiles/util.dir/PathUtil.cpp.o' failed
make[2]: *** [src/util/CMakeFiles/util.dir/PathUtil.cpp.o] Error 1
CMakeFiles/Makefile2:426: recipe for target 'src/util/CMakeFiles/util.dir/all' failed
make[1]: *** [src/util/CMakeFiles/util.dir/all] Error 2
Makefile:151: recipe for target 'all' failed
make: *** [all] Error 2

```
**To Reproduce**
try to compile on ubuntu 18.04.5 LTS

**Expected behavior**
compilation is ok

--- a/cmake/find/FindFilesystem.cmake
+++ b/cmake/find/FindFilesystem.cmake
@@ -187,7 +187,8 @@ if (CXX_FILESYSTEM_HAVE_FS)
 
         int main() {
             auto cwd = @CXX_FILESYSTEM_NAMESPACE@::current_path();
-            return cwd.string().size();
+            auto relative = @CXX_FILESYSTEM_NAMESPACE@::relative(cwd,cwd);
+            return relative.string().size();
         }
     ]] code @ONLY)
 

